debian/wazo-dist-upgrade.postinst

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+# NOTE(fblackburn): cannot use dh_installalternatives until we support buster (we need debhelper 13)
+# Automatically added by dh_installalternatives/13.3.4
+if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ]; then
+        update-alternatives --install /usr/bin/wazo-dist-upgrade-bullseye wazo-dist-upgrade-bullseye /usr/bin/wazo-dist-upgrade-bullseye-ce 25
```

Comment:
Do we do 4 spaces or tabs in bash scripts? This one has 8 spaces and .prerm has a tab.

Reply:
Yes fixed